py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/config_base.py

```diff
@@ -19,6 +19,24 @@ class ConfigType(Enum):
     BUS_VEHICLE_POSITIONS = auto()
     VEHICLE_COUNT = auto()
 
+    @classmethod
+    def from_filename(cls, filename: str):
+        if 'mbta.com_realtime_Alerts_enhanced' in filename:
+            return cls.RT_ALERTS
+        if 'mbta.com_realtime_TripUpdates_enhanced' in filename:
+            return cls.RT_TRIP_UPDATES
+        if 'mbta.com_realtime_VehiclePositions_enhanced' in filename:
+            return cls.RT_VEHICLE_POSITIONS
+        if 'com_prod_TripUpdates_enhanced' in filename:
+            return cls.BUS_TRIP_UPDATES
+        if 'com_prod_VehiclePositions_enhanced' in filename:
+            return cls.BUS_VEHICLE_POSITIONS
+        if 'net_vehicleCount' in filename:
+            return cls.VEHICLE_COUNT
+
+        raise Exception("Bad Configuration Type from filename %s" % filename)
+
+
```

Comment:
i brought this back when it was clear we'd want it for batching.

i didn't want to create an entire `Configuration` object to figure out which type a file was.